debian/copyright

```diff
@@ -0,0 +1,33 @@
+Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Upstream-Name: llmnrd
+Source: https://github.com/tklauser/llmnrd
+
+Files: *
+Copyright: 2016 Tobias Klauser <tklauser@distanz.ch>
+License: GPL-2.0+
+
+Files: debian/*
+Copyright: 2016 Tobias Klauser <tklauser@distanz.ch>
+License: GPL-2.0+
+
+License: GPL-2.0+
```

Comment:
Please note that llmnrd is licensed under GPL-v2 explicitely, _not_ any later version.